Dockerfile

```diff
@@ -24,9 +24,12 @@ WORKDIR /matterwick/
 
 RUN  apk update && apk add ca-certificates
 
+
 COPY --from=build /matterwick/build/_output/bin/matterwick /matterwick/matterwick
 COPY --from=build /matterwick/build/bin /usr/local/bin
 
+COPY ./templates /matterwick/templates
```

Comment:
you can copy from 
`COPY --from=build /matterwick/templates /matterwick/templates`